release/build_pip_package.sh

```diff
@@ -16,44 +16,58 @@
 set -e
 set -x
 
+# Pick the Python that TFQ/TensorFlow used during configure/build.
+# Order: explicit env -> 3.11 -> python3
+PY="${PYTHON_BIN_PATH:-}"
+if [[ -z "$PY" ]]; then
+  if command -v python3.11 >/dev/null 2>&1; then
```

Comment:
We should probably not force python3.11 unconditionally in the build scripts. It would probably be best to use `python3` here, and leave it to the calling environment to make sure the version of Python meets a minimum.

(And now that this comes up, I can't find a place in the code where a minimum version is set. It would be at least in the `setup.py` file, but it doesn't have anything. I'll open an issue.)